web-local/src/lib/svelte-query/actions.ts

```diff
@@ -47,7 +47,12 @@ export function useAllNames(instanceId: string) {
   );
 }
 
-export function isDuplicateName(name: string, names: Array<string>) {
+export function isDuplicateName(
+  name: string,
+  fromName: string,
+  names: Array<string>
+) {
+  if (name.toLowerCase() === fromName.toLowerCase()) return false;
   return names.findIndex((n) => n.toLowerCase() === name.toLowerCase()) >= 0;
```

Comment:
Any reason why we are not just using 
```
return names.findIndex((n) => n === name) >= 0;
```  
This wouldn't require adding extra params to `isDuplicateName`

Reply:
The method is `isDuplicateName` right, so shouldn't it return false if we want to allow entities with same letters but different case? 
So let's say there are two models named `modelA` and `modelB`. Cases this should satisfy is -
`modelA` -> `ModelA`
`modelA` -> `ModelB`

With the current implementation `modelA` -> `ModelB` is not possible.

Reply:
Ok some background, file system on mac is case insensitive. So `ModelA` and `modela` will have to be the same entitiy. So we shouldnt allow for replacing another entity but we should allow rename to different case.

In your example
modelA -> ModelA - Should be allowed, return true
modelA -> ModelB - Should not be allowed, return false